source/manual/upload-asset-to-whitehall.html.md

```diff
@@ -0,0 +1,50 @@
+---
+owner_slack: "#2ndline"
+title: Upload an asset to whitehall
+section: Assets
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-10-20
+review_in: 6 months
+---
+
+Sometimes publishers ask us to help them upload very large attachments to
+documents in whitehall because they see file size errors when they try to upload
```

Comment:
In our case the user was getting a whitehall timeout (after about 10 seconds), possibly because the 80mb zip filed unzipped to a 2gb CSV.

Reply:
👍 I've changed the wording to "timeout".